docs/source/install.rst

```diff
@@ -0,0 +1,16 @@
+Installation
+============
+
+The easiest way to install pynucastro is via pip:
+
+.. prompt:: bash
```

Comment:
do we need to include `sphinx-prompt` extension in conf.py?

Reply:
yes.  I'll fix that

Reply:
done